### PR TITLE
Text Analytics September Release note, OutOfBand

### PR DESCRIPTION
**Pull request by mssfang**
[Out of Bound] 
TA release a preview version in 09/17/2020
This is an out of bound release.

### CODE REVIEW

File: releases/2020-09/java.md [hunk 1]
```diff
@@ -586,6 +593,22 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
...
+  - `offset` is the offset of the text from the start of the document
+- Updated Text Analytics SDK's default service API version to `v3.1-preview.2` from `v3.0`.
+- Added support for Personally Identifiable Information(PII) entity recognition feature.
+  To use this feature, you need to make sure you are using the service's v3.1-preview.1 API.
+- Added support for the Opinion Mining feature. To use this feature, you need to make sure you are using the 
+service's v3.1-preview.1 and above API. To get this support pass `includeOpinionMining` as `true` in 
+`AnalyzeSentimentOptions` when calling the sentiment analysis endpoints.
+- Add property `bingEntitySearchApiId` to the `LinkedEntity` class. This property is only available for v3.1-preview.2
+and up, and it is to be used in conjunction with the Bing Entity Search API to fetch additional relevant information
+about the returned entity.
```
Comment (mssfang) [inline releases/2020-09/java.md:610]:
```suggestion
- Updated Text Analytics SDK's default service API version to `v3.1-preview.2` from `v3.0`.
- Added support for Personally Identifiable Information(PII) entity recognition feature.
  To use this feature, you need to make sure you are using the service's v3.1-preview.1 API.
- Added support for the Opinion Mining feature. To use this feature, you need to make sure you are using the 
service's v3.1-preview.1 and above API. To get this support pass `includeOpinionMining` as `true` in 
`AnalyzeSentimentOptions` when calling the sentiment analysis endpoints.

```

File: releases/2020-09/java.md [hunk 2]
```diff
@@ -586,6 +593,17 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
...
+
+#### New features
+- Updated Text Analytics SDK's default service API version to `v3.1-preview.2` from `v3.0`.
+- Added support for Personally Identifiable Information(PII) entity recognition feature.
+  To use this feature, you need to make sure you are using the service's v3.1-preview.1 API.
+- Added support for the Opinion Mining feature. To use this feature, you need to make sure you are using the 
+service's v3.1-preview.1 and above API. To get this support pass `includeOpinionMining` as `true` in 
+`AnalyzeSentimentOptions` when calling the sentiment analysis endpoints.
+
+
```
Comment (samvaity) [inline releases/2020-09/java.md:606]: nit extra line.

**Review by samvaity — APPROVED** [review]

File: releases/2020-09/java.md (hunk 2 above)
Comment (mssfang) [inline releases/2020-09/java.md:606]:
```suggestion

```

File: releases/2020-09/java.md [hunk 3]
```diff
@@ -586,6 +593,15 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
...
 
+### Azure Text Analytics ([Changelog](https://github.com/Azure/azure-sdk-for-java/blob/azure-ai-textanalytics_5.1.0-beta.1/sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md#510-beta1-2020-09-17))
+
+#### New features
+- Updated Text Analytics SDK's default service API version to `v3.1-preview.2` from `v3.0`.
+- Added support for Personally Identifiable Information(PII) entity recognition feature.
+  To use this feature, you need to make sure you are using the service's v3.1-preview.1 API.
+- Added support for the Opinion Mining feature. To use this feature, you need to make sure you are using the 
+service's v3.1-preview.1 and above API. To get this support pass `includeOpinionMining` as `true` in 
+`AnalyzeSentimentOptions` when calling the sentiment analysis endpoints.
```
Comment (mssfang) [inline releases/2020-09/java.md:604]:
```suggestion
`AnalyzeSentimentOptions` when calling the sentiment analysis endpoints.

```

**Review by alzimmermsft — CHANGES_REQUESTED** [review]
Given this is an out of band release the release notes should go in the October release notes.

**Comment by samvaity** [comment]
@alzimmermsft Updated policy - https://github.com/Azure/azure-sdk/pull/1828/files#diff-c90d9c7056858a710b93ef7d85188b0cR60

**Comment by alzimmermsft** [comment]
> 
> 
> > Given this is an out of band release the release notes should go in the October release notes.
> 
> @alzimmermsft Updated policy - https://github.com/Azure/azure-sdk/pull/1828/files#diff-c90d9c7056858a710b93ef7d85188b0cR60

Ah, didn't see this, disregard my last comment than.

**Review by alzimmermsft — APPROVED** [review]

**Pull request merged**